src/main/java/com/styra/opa/OPAClient.java

```diff
@@ -92,6 +97,10 @@ public OPAClient(OpaApiClient client) {
         this.sdk = client;
     }
 
+    private Map<String, java.lang.Object> defaultInput() {
```

Comment:
Can we make default "undefined"?

Reply:
I'm not sure that's representable in Java? I could probably make it null.

Reply:
Hmm, yeah the `Input` object that Speakeasy generates for `Input` does not allow you to construct one that is null or undefined, which is why I did the empty object thing. I do think that undefined would be better if we can find a way. Lets discuss in planning today.

Reply:
I guess what I mean is can we -- if no input is provided -- use ExecutePolicy instead of ExecutePolicyWithInput from the generated methods?

Reply:
done in d9e72e5c3214d28bf73fad24417205cb03cbe939